### ISSUE TITLE
[Research] OUI Compliance audit of `src/core/public/overlays`

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4115]
Tracking issue for OUI compliance audit of the `src/core/public/overlays` folder.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

Note that this is a core service, which may be involved with the implementation and inclusion of OUI styles and themes. Since this is a core service, the conclusion of this audit might actually not involve any additional work.

<details>
  <summary>Related files</summary>

size (bytes) | file | notes
---|---|---
132 | [src/core/public/overlays/banners/_banners_list.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/overlays/banners/_banners_list.scss) | 
87 | [src/core/public/overlays/_mount_wrapper.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/overlays/_mount_wrapper.scss) | 
54 | [src/core/public/overlays/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/overlays/_index.scss) | 
26 | [src/core/public/overlays/banners/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/overlays/banners/_index.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.
